config/database.yml

```diff
@@ -11,14 +11,12 @@ default: &default
 
 development:
   <<: *default
-  database: education_development
```

Comment:
But in dev I can't really set two different env vars based on running dev vs test. Wasn't that annoying when you were building this?